modules/test/playwright/tests/dynamic-data-mapping-form-web/main/formSubmission.spec.ts

```diff
@@ -52,7 +52,7 @@ test.describe('Manage forms through submission page', () => {
 		browser,
 		virtualInstancesPage,
 	}) => {
-		test.slow();
+		test.setTimeout(360000);
```

Comment:
Hey @sauloviniciusf, I think this is too much. `test.slow();` should suffice. We set 360000 locally because the machine is too slow.